### ISSUE TITLE
Slight incompatibility with 'Carry On' mod

### ISSUE THREAD

**Issue by FinnTheCookie** [refinedmods/refinedstorage2#456]
### Describe the bug

Picking up the Creative Controller with the 'Carry On' mod in Survival makes it unable to be placed down, instead duplicating the block every time it is placed down. For further context, I play in a pack that doesn't have power, so I modified the regular controller recipe to make the Creative Controller so I wouldn't have to worry about power.

It is pretty much impossible to set down, and after trying to die and respawn, I was unable to respawn, at all. Even after deleting my player.dat file. The solution I found was to disable Carry On, reload the world, then re-enable it after.

### How can we reproduce this bug or crash?

1. Obtain a Creative Controller block
2. Place it down
3. Try to pick it up using the Carry On mod (Crouch + Right Click by default)

### What Minecraft version is this happening on?

Minecraft 1.20.1

### What Forge or Fabric version is this happening on?

0.14.24

### What Refined Storage version is this happening on?

v2.0.0-milestone.3.2

**Comment by raoulvdberge** [comment]
Thank you for reporting. Could you try moving some other RS2 blocks with Carry On and tell me how it works or if anything else is broken? Thanks!

**Comment by FinnTheCookie** [comment]
> Thank you for reporting. Could you try moving some other RS2 blocks with Carry On and tell me how it works or if anything else is broken? Thanks!

I tried a few more, it seems most blocks are affected, mainly those with an interface.
The ones I tried are the Controller (non-Creative version), Grid, Crafting Grid, External Storage, Interface, Disk Drive, Detector, 1K Storage Block, Network Transmitter & Network Receiver. 
The only one I tested that wasn't affected was the Machine Casing, but it wasn't able to be picked up by Carry On anyways.
